prove/Develop02/Journal.cs

```diff
@@ -1,40 +1,97 @@
+using System.ComponentModel;
+using System.IO;
 public class Journal
     {
         //Class variables
-        string _userName;
+        public string _userName;
         
         //Class for entries//
-        public class UserEntry
-        {
-
-        }
+        public List<Entry> _userEntry;
+        public string _newEntry;
 
         //Constructor//
-        public Journal()
+        public Journal(string userName)
         {
+            _userName = userName;
+            _userEntry = new List<Entry>();
         }
 
         //Functions//
-        //Get user name//
-        static string GetUserName()
-        {
-            Console.WriteLine("What is the user name? ");
-            string _userName = Console.ReadLine();
-            return _userName;
-        }
 
         //Get User Entry//
 
-        static string GetUserEntry()
+        public void NewEntry()
+        {
+            string prompt = RandomPrompt();
+            string response = Console.ReadLine();
+            DateTime intDate = DateTime.Now;
+            string currentDate = intDate.ToShortDateString();
+            Entry entry = new Entry(currentDate,prompt,response);
+            _userEntry.Add(entry);  
+        }
+        public void DisplayAll()
         {
 
+            foreach (var i in _userEntry)
+            {
+                Console.WriteLine(i.EntireEntry());
+            }
+
         }
-        static void DisplayAll()
+        public void SaveJournal()
         {
-            //work on this after working on saving data//
-            foreach (string entry in Entry)
-                {
+            string fileName = _userName + "journal.txt";
 
+            using (StreamWriter outputFile = new StreamWriter(fileName))
+            {
+                foreach (var i in _userEntry)
+                {
+                    outputFile.WriteLine(i.EntireEntry());
                 }
+
+
+            }
+        }
+
+        public void LoadJournal()
+        {
+            string filename = _userName + "journal.txt";
+            string[] lines = System.IO.File.ReadAllLines(filename);
+
+            foreach (string line in lines)
+            {
+                
+                string[] parts = line.Split(",");
```

Comment:
You have a problem that this misaligns if you have commas in prompts or responses.  For example, one of your prompts is:
"If I had one thing I could do over today, what would it be?"
This is going to be split to use the first half as the prompt and the second half as the entry and so it will ignore anything the user enters when it tries to load.
A good way to address this is to use a less common separator.  You can also use a string as the separator.  For example: "~|~"